app/scripts/lib/app-start.js

```diff
@@ -463,7 +466,8 @@ function (
           fxaClient: this._fxaClient,
           marketingEmailClient: this._marketingEmailClient,
           assertion: this._assertionLibrary,
-          storage: this._getStorageInstance()
+          storage: this._getStorageInstance(),
+          uuid: this._uuid
```

Comment:
naming nit: to me, "uuid" just reads "universally unique identifier" and doesn't give me any clues as to what this value is supposed to be used for.  Does it make sense to name it something more meaningful like "session id" or "unique user id"?  Not sure I have a good suggestion, but "uuid" just seems a little too generic.